kotlin/mobilesdkrs/build.gradle

```diff
@@ -116,8 +116,17 @@ publishing {
                 password = System.getenv("GITHUB_TOKEN")
             }
         }
+        mavenLocal()
```

Comment:
NOTE: Adding `mavenLocal` repository here, which I was able to check is publishing to `~/.m2/` local repository.

TODO: need to add the following to the README for publishing to maven local:

```bash
VERSION=SEMVER_VERSION_HERE ./gradlew publishDebugPublicationToMavenLocal
```

Reply:
See #58